src/frontend/src/pages/SettingsPage/Settings.tsx

```diff
@@ -8,9 +8,57 @@ import { useAuth } from '../../hooks/auth.hooks';
 import PageTitle from '../../layouts/PageTitle/PageTitle';
 import PageBlock from '../../layouts/PageBlock';
 import UserSettings from './UserSettings/UserSettings';
-import { Alert, Grid, Switch, FormGroup, FormControlLabel } from '@mui/material';
+import { Alert, Grid, Switch, FormGroup, FormControlLabel, SwitchProps, styled } from '@mui/material';
 import LoadingIndicator from '../../components/LoadingIndicator';
 
+const NERSwitch = styled((props: SwitchProps) => (
```

Comment:
this is very elaborate! also needs a little right margin and less left margin (just from me looking at it and not reading this giant thing)

Reply:
yeah lol, a decent bit of this was ripped/ repurposed from the mui docs; I'll mess with the margin, the left padding was that so it doesn't overlap with the organization name when you make the browser width smaller